metrics/share-of-voice.md

```diff
@@ -0,0 +1,56 @@
+# Share of Voice
+
+Can we communicate the value of our community as a comparison to something of value by our supporting business?
```

Comment:
This is one alternative that makes more sense to me. We can discuss.

For sure, we want to put `Question:` at the beginning.
```suggestion
Question: If a supporting business values the "share of voice" metric, how can the value of a community be expressed in those terms?
```